justfile

```
@@ -43,6 +45,28 @@ build-target target *args:
     cargo build -p tinybmp --target {{target}} {{args}}
     cargo build -p tinybmp --target {{target}} --all-features {{args}}
 
+# Cross compiles embedded-graphics tinytga and tinybmp for all targets
+build-targets *args:
+    #!/usr/bin/env bash
+    set -e
+
+    for target in {{targets}}; do just build-target $target {{args}}; done
```

Bash shouldn't be required for this task:
```suggestion
    for target in {{targets}}; do just build-target $target {{args}}; done
```